eth/stagedsync/stage_txlookup.go

```diff
@@ -89,6 +93,7 @@ func TxLookupTransform(logPrefix string, tx kv.RwTx, startKey, endKey []byte, qu
 	return etl.Transform(logPrefix, tx, kv.HeaderCanonical, kv.TxLookup, cfg.tmpdir, func(k []byte, v []byte, next etl.ExtractNextFunc) error {
 		blocknum := binary.BigEndian.Uint64(k)
 		blockHash := common.BytesToHash(v)
+		borPrefix := []byte("matic-bor-receipt-")
```

Comment:
not important:
if full key size is know - can pre-alloc it. 
`next` internally doing copy of bytes - means also can re-use pre-allocated array.